tests/console/yast2_xinetd.pm

```diff
@@ -93,8 +93,7 @@ sub run {
     wait_serial('yast2-xinetd-status-0', 180) || die "'yast2 xinetd' didn't finish";
 
     # check xinetd configuration
-    assert_script_run("systemctl show -p ActiveState xinetd.service | grep ActiveState=active");
-
+    systemctl 'show -p ActiveState xinetd.service | grep ActiveState=active';
```

Comment:
As above, not only action on service. Maybe worth adding functionality to systemctl subroutine.